.github/workflows/test-unit.yml

```diff
@@ -0,0 +1,27 @@
+name: TestSuite integration and unit tests
+
+on:
+  pull_request:
+    branches: [ "main" ]
+    paths:
+      - 'go.mod'
+      - 'go.sum'
+      - 'Makefile'
+      - '**.go'
+jobs:
+  envtest-and-unittest:
+    name: "Run 'make test'"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout lifecycle-manager
+        uses: actions/checkout@v3
+        with:
+          path: ./lifecycle-manager/
```

Comment:
Are these paths in `actions/checkout@v3` and `Run 'make test'` context should be this root already

Reply:
For me personal it is more readable if we always mention the path, but that is personal preference :D I removed it